tests/conftest.py

```diff
@@ -303,6 +303,9 @@ def script(tmpdir, virtualenv, deprecated_python):
         assert_no_temp=True,
 
         # Deprecated python versions produce an extra deprecation warning
+        # This also addresses a possible warning on old versions of Python
+        # (< 2.7.9) about the lack of an SSLContext in urllib3/requests,
```

Comment:
This isn't clear: one could think that 2.7.9 is dealt with in `deprecated_python` but currently, only 2.7 and 3.4 are.
(and if the 2.7 warning is removed, it's unlikely someone will add a special case for 2.7.9...)
I think we can drop the comment about 2.7.9 altogether.